doc/raspberrypi-64.md

```diff
@@ -29,8 +29,19 @@ $ source repos/meta-emlinux/scripts/setup-emlinux build
 
 In the build directroy, you can build image. You need to set necessary variables in the relevant .conf files.
 
+Building for Raspberry Pi 3B+:
 ```
 $ echo 'MACHINE = "raspberrypi3bplus-64"' >> conf/local.conf
+```
+
+Building for Raspberry Pi 4B:
+```
+$ echo 'MACHINE = "raspberrypi4b-64"' >> conf/local.conf
+```
+
+You also need to add the following to build image.
+
+```
 $ echo 'BBLAYERS += "${TOPDIR}/../repos/isar/meta-isar"' >> conf/bblayers.conf
```

Comment:
This bblayers setting is for using rpi-sdimg under meta-isar, but emlinux-rpi-sdimg has been applied instead since your 4B support, so the step has not been needed. Remove this description.

Reply:
I corrected it as you suggested.